style.css

```diff
@@ -1,8 +1,8 @@
 main {
   position: relative;
   width: 375px;
-  height: 812px;
-  background: #ececf0;
+  height: 5315px;
+  background: #f7f7f9;
 }
```

Comment:
- [ OPTIONAL ] :thinking: I think it would be better if you used a `100vw` for width and kept `auto` for height to make sure your website is responsive :recycle: